.github/workflows/compressed-size-action.yml

```diff
@@ -9,9 +9,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
```

Comment:
Can we change this to v4 like you did in waku? (so sorry, I should have checked this before asking for zustand&valtio.)

Reply:
yeah we can

Reply:
let's do that then. can you update all three prs?